Jetcaster/app/src/main/java/com/example/jetcaster/ui/home/category/PodcastCategory.kt

```diff
@@ -110,6 +112,15 @@ fun PodcastCategory(
                 modifier = Modifier.fillParentMaxWidth()
             )
         }
+    }
+}
+
+@Composable
+private fun EpisodeList(viewState: PodcastCategoryViewState) {
```

Comment:
We can probably just pass in the list of episodes rather than the entire view state

Reply:
Changed it to take the list of episodes.